waldo/tool/xcode/xcode.go

```diff
@@ -123,6 +137,12 @@ func (xb *XcodeBuilder) Summarize() string {
 	return summary
 }
 
+func (xb *XcodeBuilder) VerifyBuildPath(basePath string, ios *lib.IOStreams) (string, error) {
```

Comment:
Theses aren't used yet ?

Reply:
Correct. It will be used by React Native (and possibly Expo). RN relies on info in Xcode and Gradle.